### ISSUE TITLE
Question: Any plan to add official accounts? 

### ISSUE THREAD

**Issue by jakeau** [Alex2772/hackers-mc-launcher#1]
It possibly defeats the idea of your launcher but could be a nice option.

**Comment by Alex2772** [comment]
Yes, I have wanted to add, but I have no official Minecraft account for debugging.

**Comment by jakeau** [comment]
Add me on discord I can provide you one JakeAU#4161, I also have a c++ library to validate accounts sitting in one of my repos somewhere I'll see if I can find that and send it your way.
